engineering/conventions-go.md

```diff
@@ -0,0 +1,56 @@
+
+# Go Development Conventions
```

Comment:
I wish to add here some styling rules, that are not in go-vet, almost everyone is writing a new line after a "logic block", I mean:

```go
var wg sync.WaitGroup
wg.Add(1)

qux := 1
```

But some people is writing without any new line inside of a function, so will be good include this in the guide.

Reply:
Added a paragraph about it. I think we need a couple of better examples though.

Reply:
I think it might be better to write a linter that enforces these rules. We had one in previous company.